py3status/modules/coin_market.py

```diff
@@ -0,0 +1,168 @@
+# -*- coding: utf-8 -*-
+"""
+Display cryptocurrency data.
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module. A message from the site:
+        Please limit requests to no more than 10 per minute.
+        (default 600)
+    format: display format for this module (default '{format_coin}')
+    format_coin: display format for coins
+        (default '[\?color=degraded {name}] ${price_usd:.2f} {percent_change_24h}')
+    format_separator: show separator only if more than one (default ' ')
+    markets: list of supported markets https://coinmarketcap.com (default ['btc'])
+    request_timeout: time to wait for a response, in seconds (default 5)
+
+Format placeholder:
+    {format_coin} format for cryptocurrency coins
+
+format_coin placeholders:
+    {24h_volume_usd}      eg 1435150000.0
+    {available_supply}    eg 16404825.0
+    {id}                  eg bitcoin
+    {last_updated}        eg 1498135152
+    {market_cap_usd}      eg 44119956596.0
+    {name}                eg Bitcoin
+    {percent_change_1h}   eg -0.17%
+    {percent_change_24h}  eg -1.93%
+    {percent_change_7d}   eg +14.73%
+    {price_btc}           eg 1.0
+    {price_usd}           eg 2689.45
+    {rank}                eg 1
+    {symbol}              eg BTC
+    {total_supply}        eg 16404825.0
+
+    Placeholders are retrieved directly from the URL.
+    The list was harvested only once and should not represent a full list.
+
+    To print coins in different currency, replace or replicate the placeholders
+    below with a valid option (eg '{price_gbp}') to create additional placeholders:
+
+    {price_xxx}           eg 1819.67786724  # GBP
+    {24h_volume_xxx}      eg 668740360.049  # GBP
+    {market_cap_xxx}      eg 29926922616.0  # GBP
+
+    Valid options are: AUD, BRL, CAD, CHF, CNY, EUR, GBP, HKD, IDR, INR, JPY,
+    KRW, MXN, RUB, otherwise USD. Valid options must be written in lowercase.
+
+Examples:
+```
+# view coins in GBP
+coin_market {
+    format_coin = '[\?color=degraded {name}] £{price_gbp:.2f} {percent_change_24h}'
+}
+
+# view coins in GBP and USD
+coin_market {
+    format_coin = '[\?color=degraded {name}] '
+    format_coin += '£{price_gbp:.2f} ${price_usd:.2f} {percent_change_24h}'
+}
+
+# change market name color
+coin_market {
+    color_degraded = '#3DAEE9'
+}
+```
+
+@author x86kernel, lasers
+
+SAMPLE OUTPUT
+[
+    {'color': u'#FFFF00', 'full_text': u'Bitcoin'},
+    {'full_text': u' $2735.77 '},
+    {'color': '#00FF00', 'full_text': u'+2.27%'},
+]
+
+losers
+[
+    {'color': u'#FFFF00', 'full_text': u'Bitcoin'},
+    {'full_text': u' $2701.70 '},
+    {'color': '#FF0000', 'full_text': u'-0.42%'},
+]
+"""
+
+
+class Py3status:
+    cache_timeout = 600
+    format = '{format_coin}'
+    format_coin = '[\?color=degraded {name}] ${price_usd:.2f} {percent_change_24h}'
```

Comment:
You can try `limit = 10` for a while... to understand why I wanted to add colors. It can be a tad hard to parse things without them. This is to make multiple markets readability on the off chance somebody could be using them and that they don't have to configure `format` out of the box... assuming default `format` is not awful.

```
# color just works without adding anything
coin_markets {
    limit = 10
}
```
```
# color just works without adding anything
coin_markets {
    markets = ['btc', 'dash', 'eth', 'sky'] 
}
```